### PR TITLE
visibility hidden when the resolution is bellow 700px

### PR DESCRIPTION
**Pull request by xiCO2k**
hide the new message on resolutions bellow.

Right now its like this:
image

And will be like:
image

### CODE REVIEW

File: Messenger/JSNotification.js.mm [hunk 1]
```diff
@@ -51,7 +51,6 @@ bool jsinit(size_t argc, const JSValueRef argv[], JSValueRef* exc) {
       }
     }
 
-    notif.soundName = NSUserNotificationDefaultSoundName;
```
Comment (rsms) [inline Messenger/JSNotification.js.mm:54]: Why this change? Seems unrelated to hiding the "new message" icon.

File: website/app/main.js [hunk 2]
```diff
@@ -395,7 +395,10 @@
     // #146 Fix for vertical scrollbar appearing when mouse plugged in
     var css = document.createElement('style');
     css.type = 'text/css';
+
     var style = 'body { overflow: hidden; }'
+    style += '@media (max-width: 700px) { [aria-label="New Message"] { visibility: hidden; } }'
```
Comment (rsms) [inline website/app/main.js:400]: This will only work for people with en_US locale on Facebook. Not even en_EN (where it's most likely "New message".) You might want to consider a selector (or some other way of finding the element) that doesn't rely on user settings. You can try things out by chaning your language under account settings on Facebook.

**Review by rsms — CHANGES_REQUESTED** [review]

File: website/app/main.js [hunk 3]
```diff
@@ -395,7 +395,10 @@
...
+
```
Comment (majorjammer) [inline website/app/main.js:401]: 11

**Comment by majorjammer** [comment]
11

**Comment by majorjammer** [comment]
11
